lib/json_schema_form/umm_form.rb

```diff
@@ -298,6 +298,36 @@ def accordion_id
       top_key.underscore.dasherize
     end
   end
+
+  def field_title_differs_from_accordion_title?
```

Comment:
minor quibble and semantic, but I think using `is` or `not` would be more accurate that `differs_from` in the method name.